### PR TITLE
Expose both original and thumbnail logo

### LINKED ISSUE #824: Unable to retrieve full size organization logo from API
The API returns an URL to a resized image (100x100):
https://www.data.gouv.fr/api/1/organizations/54da1f52c751df59f7467389/

The regular organization page display the full size image:
https://www.data.gouv.fr/fr/organizations/grand-poitiers-open-data/

Consequently we are forced to display a low resolution image in our geo-specific portal:
https://inspire.data.gouv.fr/datasets/e80d3d24c0540868a5656ab8bb91b66af24b80e4

### PR DESCRIPTION
**Pull request by noirbizarre**
fix #824

### CODE REVIEW

File: udata/core/organization/api_fields.py [hunk 1]
```diff
@@ -71,7 +71,10 @@
     'page': fields.UrlFor(
         'organizations.show', lambda o: {'org': o},
         description='The organization page URL', readonly=True),
-    'logo': fields.ImageField(description='The organization logo URLs'),
+    'logo': fields.ImageField(original=True,
+        description='The organization logo URL'),
+    'logo_thumbnail': fields.ImageField(size=150,
+        description='The organization logo thumbnail (cropped) URL'),
```
Comment (davidbgk) [inline udata/core/organization/api_fields.py]: Add `150` in the description? (same to the `logo` of `org_ref_fields`)

File: udata/core/organization/api_fields.py [hunk 2]
```diff
@@ -71,7 +71,10 @@
     'page': fields.UrlFor(
         'organizations.show', lambda o: {'org': o},
         description='The organization page URL', readonly=True),
-    'logo': fields.ImageField(description='The organization logo URLs'),
+    'logo': fields.ImageField(original=True,
+        description='The organization logo URL'),
+    'logo_thumbnail': fields.ImageField(size=150,
```
Comment (davidbgk) [inline udata/core/organization/api_fields.py]: Why `150` given that we have `100` above for the logo of `org_ref_fields`? Magic number => descriptive var?

**Comment by noirbizarre** [comment]
Changed

**Comment by davidbgk** [comment]
See my general comment too.

**Comment by noirbizarre** [comment]
Done 👍

**Comment by noirbizarre** [comment]
Size added :wink:

**Pull request merged**